pkg/server/storage_api/ranges_test.go

```diff
@@ -133,21 +133,58 @@ func TestRangeResponse(t *testing.T) {
 	defer log.Scope(t).Close(t)
 	defer kvserver.EnableLeaseHistoryForTesting(100)()
 	srv := serverutils.StartServerOnly(t, base.TestServerArgs{
-		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(110018),
+		DefaultTestTenant: base.TestTenantProbabilistic,
 	})
 	defer srv.Stopper().Stop(context.Background())
 	ts := srv.ApplicationLayer()
 
+	t.Logf("value of StartedDefaultTestTenant %v", srv.StartedDefaultTestTenant())
+
```

Comment:
When unit tests are run they print the mode of deployment

Reply:
Have removed this print statement from the test code.

I looked at the logs and I can see a log line saying something like `I250130 11:33:23.511248 2408 1@server/tenant.go:301  [n1] 194  server starting for tenant "10"` when the tenant was enabled. However, I didn't find any specific logs w.r.t. system tenant starting. Can you point me to those logs for my reference? Or is it just the absence of this very log statement points to the fact that the test is started with a system tenant? Thanks!